lib/Map/TableColumn.js

```diff
@@ -164,7 +164,9 @@ var TableColumn = function(name, values, options) {
 
     this._formattedValues = getFormattedValues(this);
 
-    knockout.track(this, ['_type']);  // so that TableStructure can change columnsByType if type changes.
+    // Track _type so that TableStructure can change columnsByType if type changes.
+    // Track _values so that charts can update live with new data.
+    knockout.track(this, ['_type', '_values']);
```

Comment:
Here are some tests with a 55,000-line region-mapped csv and a stopwatch on my computer, with Cesium:
```
             load     change var to First Letter
newui        8.5s     4.5s
newui_poll   7.4s     4.3s
```
There's probably a wide margin of error on this :-), but at least it seems ok. Do you have any other test files in mind?